### PR TITLE
ospp/2022: update descriptions for the AOSC Wiki project

### PR DESCRIPTION
**Pull request by OriginCode**
Update descriptions and translations (en_US) for "Add Missing Documents and Update Existing Documents on AOSC Wiki" project.

### CODE REVIEW

File: content/community/ospp/2022-projects.md [hunk 1]
```diff
@@ -79,21 +79,22 @@ AOSC's package information site is now severely disconnected from the current AO
...
 
 There are many AOSC projects missing documents on AOSC Wiki, and many existing documents are outdated. This project will update existing documents and add undocumented project documents to AOSC Wiki.
 
-- Difficulty: Advanced [Temporary]
+- Difficulty: Advanced [Pending]
 - Mentor: Kaiyang Wu
 - Mentor contact: origincode@aosc.io
 - Project Requirements:
-  - Add documents for the projects in the work ticket, make sure the English contents are the latest if possible
+  - Communicate with community contributors and collect entries for repair or writing, and develop a work plan.
```
Comment (MingcongBai) [inline content/community/ospp/2022-projects.md:90]: - "entries" => "Wiki entries"
- "For improve to writing" => "to repair or create,"
- "develop" => "draft"

File: content/community/ospp/2022-projects.md [hunk 2]
```diff
@@ -79,21 +79,22 @@ AOSC's package information site is now severely disconnected from the current AO
...
+  - Repair or write entries on the work plan to ensure that the English pages on the Wiki are up to date.
```
Comment (MingcongBai) [inline content/community/ospp/2022-projects.md:91]: - "Repair or write entries" => "Improve or create Wiki entries"
- "the English pages on the Wiki are up to date" => "the Wiki entries reflect up-to-date information"

File: content/community/ospp/2022-projects.md [hunk 3]
```diff
@@ -79,21 +79,22 @@ AOSC's package information site is now severely disconnected from the current AO
...
 - Technical Requirements:
-  - Ability to write in formal English, familiarity with computer/Linux-related terminologies.
-  - Familiarity with Markdown format.
-  - Familiarity with searching documents for open-source projects (manpage, README etc.).
-  - (If possible) be familiar with Rust, Python etc.
-  - Essential familiarity with the current projects maintained by AOSC and the projects used by AOSC OS.
+  - Adequate English writing skills and familiarity with computer and Linux related terminology.
```
Comment (MingcongBai) [inline content/community/ospp/2022-projects.md:93]: Proficiency in English writing and familiarity with Linux-related terminologies.

File: content/community/ospp/2022-projects.md [hunk 4]
```diff
@@ -79,21 +79,22 @@ AOSC's package information site is now severely disconnected from the current AO
...
+  - Familiarity with Markdown and formatting specifications.
```
Comment (MingcongBai) [inline content/community/ospp/2022-projects.md:94]: What do you mean by "formatting specifications"?

File: content/community/ospp/2022-projects.md [hunk 5]
```diff
@@ -79,21 +79,22 @@ AOSC's package information site is now severely disconnected from the current AO
...
+  - Familiarity with various kinds of open-source project documentation to find (such as manpage, README, etc.).
```
Comment (MingcongBai) [inline content/community/ospp/2022-projects.md:95]: I failed to understand you. Neither is the Chinese version clear.

File: content/community/ospp/2022-projects.md [hunk 6]
```diff
@@ -79,21 +79,22 @@ AOSC's package information site is now severely disconnected from the current AO
...
+  - Familiarity with Rust, Python, etc. as much as possible.
```
Comment (MingcongBai) [inline content/community/ospp/2022-projects.md:96]: "As much as possible" not reflected in Chinese version - also, why this requirement?

File: content/community/ospp/2022-projects.md [hunk 7]
```diff
@@ -79,21 +79,22 @@ AOSC's package information site is now severely disconnected from the current AO
...
+  - Basic understanding of the projects currently maintained by AOSC / currently used by AOSC OS.
```
Comment (MingcongBai) [inline content/community/ospp/2022-projects.md:97]: Knowledge of current AOSC projects and components available for AOSC OS.

File: content/community/ospp/2022-projects.zh.md [hunk 8]
```diff
@@ -112,13 +112,14 @@ AOSC Wiki 目前尚缺失大量社区项目的文档，以及存在大量过期
...
 - 导师联系方式：origincode@aosc.io
 - 项目产出要求：
-  - 为工作单内的项目或功能更新或新增内容，尽可能保证英文版本的 Wiki 内容为最新。
+  - 与社区贡献者沟通并收集修缮或撰写的条目，制定工作计划。
+  - 按工作计划修缮或撰写条目，以确保 Wiki 站点中的英文页面内容最新。
 - 项目技术要求：
   - 有足够英语写作能力，熟悉计算机、Linux 相关术语。
   - 熟悉 Markdown 的编写和格式规范。
   - 熟悉各类开源项目文档的查找（如 manpage、README 等）。
   - 尽可能熟悉 Rust、Python 等语言的代码。
```
Comment (MingcongBai) [inline content/community/ospp/2022-projects.zh.md:120]: Probably an unnecessary requirement.

**Review by MingcongBai — CHANGES_REQUESTED** [review]

File: content/community/ospp/2022-projects.md (hunk 4 above)
Comment (OriginCode) [reply content/community/ospp/2022-projects.md:94]: "=> Familiarity with Markdown" ?

File: content/community/ospp/2022-projects.md (hunk 5 above)
Comment (OriginCode) [reply content/community/ospp/2022-projects.md:95]: => 熟悉查阅各类开源项目文档（如 manpage、README 等）。

File: content/community/ospp/2022-projects.md (hunk 6 above)
Comment (OriginCode) [reply content/community/ospp/2022-projects.md:96]: For situations when there's no official document for the project.

File: content/community/ospp/2022-projects.md (hunk 7 above)
Comment (OriginCode) [reply content/community/ospp/2022-projects.md:97]: Or, "Basic understanding of the projects currently maintained by AOSC." ?

File: content/community/ospp/2022-projects.zh.md (hunk 8 above)
Comment (OriginCode) [reply content/community/ospp/2022-projects.zh.md:120]: Which one?

File: content/community/ospp/2022-projects.md (hunk 5 above)
Comment (MingcongBai) [reply content/community/ospp/2022-projects.md:95]: Sure.

File: content/community/ospp/2022-projects.md (hunk 6 above)
Comment (MingcongBai) [reply content/community/ospp/2022-projects.md:96]: You are confusing me even more. Just remove it.

File: content/community/ospp/2022-projects.md (hunk 7 above)
Comment (MingcongBai) [reply content/community/ospp/2022-projects.md:97]: Then it doesn't mean the same thing.

File: content/community/ospp/2022-projects.zh.md (hunk 8 above)
Comment (MingcongBai) [reply content/community/ospp/2022-projects.zh.md:120]: The line I'm pointing to, of course.

File: content/community/ospp/2022-projects.md (hunk 7 above)
Comment (OriginCode) [reply content/community/ospp/2022-projects.md:97]: > Then it doesn't mean the same thing.

I decided to go with "Basic understanding of the projects currently maintained by AOSC.".

File: content/community/ospp/2022-projects.md (hunk 6 above)
Comment (OriginCode) [reply content/community/ospp/2022-projects.md:96]: > You are confusing me even more. Just remove it.

OK.

File: content/community/ospp/2022-projects.md (hunk 7 above)
Comment (MingcongBai) [reply content/community/ospp/2022-projects.md:97]: I'm trying to remove all passive voice, and you should too.

File: content/community/ospp/2022-projects.md (hunk 4 above)
Comment (MingcongBai) [reply content/community/ospp/2022-projects.md:94]: Sure.

File: content/community/ospp/2022-projects.md (hunk 7 above)
Comment (OriginCode) [reply content/community/ospp/2022-projects.md:97]: > I'm trying to remove all passive voice, and you should too.

=> Knowledge of current AOSC projects.

Also, any advice for the Chinese version? I came up with "了解 AOSC 的项目。".

File: content/community/ospp/2022-projects.zh.md (hunk 8 above)
Comment (OriginCode) [reply content/community/ospp/2022-projects.zh.md:120]: > The line I'm pointing to, of course.

To be deleted.

**Review by MingcongBai — APPROVED** [review]

**Pull request merged**